test/commands/commit.spec.js

```diff
@@ -142,6 +144,8 @@ describe('commit command', () => {
         getEmojis.mockResolvedValue(stubs.gitmojis)
         mockProcess.mockProcessExit()
         process.argv[3] = stubs.argv
+        execa.mockReturnValueOnce(Promise.resolve(stubs.gitAbsoluteDir))
+        fs.existsSync.mockReturnValueOnce(false).mockReturnValueOnce(false)
```

Comment:
Why we need to add those mocks inside this test ?

Reply:
to pass the is `cancelIfRebasing` test